src/Settings/AppSettings.cc

```diff
@@ -192,33 +216,39 @@ DECLARE_SETTINGSFACT_NO_FUNC(AppSettings, qLocaleLanguage)
         FactMetaData*   metaData            = _qLocaleLanguageFact->metaData();
         QStringList     rgOriginalStrings   = metaData->enumStrings();
```

Comment:
I guess the original values aren't needed anymore, unless you want to check for changes

Reply:
Oops, correct. Those are gone now from the json.